### ISSUE TITLE
Imputation before filtering

### ISSUE THREAD

**Issue by ajcammack** [ndcn/soda-ndcn#141]
The imputation really helps for class and total normalization because without it, the non-missing values are inflated when there are a lot of missing values. But then I think it might be nice to have the option to run the statistics excluding the imputed values, since they are not real. 

Alternatively, could impute and normalize first and then filter to remove species with imputed values entirely. This would require imputed values to be considered "NA" for the process of filtering only. Note that then normalized values would not add up to 1 since normalization would be done before filtering (but I think this is okay?)

**Comment by ajcammack** [comment]
Should say this is a minor issue and the imputation overall is really good and helps a lot with normalization

**Comment by dolivierj** [comment]
There's actually already a 'Impute before filtering' switch below 'Apply imputation'! Turning it on will apply imputation and then filter, off, it will impute only after filtering. Let me know if this is what you wanted before I close the issue!
